### ISSUE TITLE
[FEATRE REQUEST] Optional Prefix

### ISSUE THREAD

**Issue by fuhlig** [baerkins/vscode-fractal-folder#2]
Proposal:
When creating a new folder, an optional prefix can be provided for ordering (see [documentation](https://fractal.build/guide/components/notes.html#the-readme-md-file)).

This should be only added to the folder name, not the files itself

```
05-modal
--- modal.hbs
--- modal.config.js

10-card
--- card.hbs
--- card.config.js 
```

Thanks in advance for considering : )

**Comment by baerkins** [comment]
This seems useful. For the logic, any leading numbers and a following dash or underscore could be removed from the name. Something to cover:

`01-name -> 01-name/name.hbs`
`2_name -> 2_name/name.hbs`

But not `name-01`, which would result in `name-01/name-01.hbs`

Another option would be to add a custom regex in the options that would be ran when creating the file names. That might keep things less opinionated.

Would love to hear feedback!

**Comment by fuhlig** [comment]
Added the wrong docs reference. Here's the correct link:
https://fractal.build/guide/core-concepts/naming.html#ordering-and-hiding

_Items can be ordered by prefixing file names with a **two digit number** (with leading zero, if required) **followed by a hyphen**._

An additional prompt could open after entering the name to enter the optional prefix.

**Comment by baerkins** [comment]
Got it - thanks! Will add a regex for leading numbers with a dash.

Cheers

**Comment by baerkins** [comment]
Closed in 069bc48
